CHANGES

```diff
@@ -27,6 +27,7 @@ Version 8.14
 - Added bulk "Resolve in Next Release" to stream actions.
 - Various visual improvements to notifications, including the addition of
   transactions.
+- Added feature signals, such as first time release tracking is enabled.
```

Comment:
Move this to 8.15.

Could also honestly mark this as `#nochanges`. This has no implication on public consumers.